docs/UserGuide.md

```diff
@@ -3,7 +3,7 @@ layout: page
 title: User Guide
 ---
 
-RealEstatePro is a desktop app for managing contacts, optimized for real estate agents to manage their client's contacts and sales of HDB properties.
+RealEstatePro is a desktop app for managing contacts, optimized for real estate agents to manage their client's contacts and sales of properties.
```

Comment:
Better to specify HDB properties. Refer to #137

Reply:
Yeap but I was thinking of condominiums as well. How about stating HDB & Condominium properties?